downstream/modules/eda/proc-eda-delete-rulebook-activations-with-cont-tokens.adoc

```diff
@@ -0,0 +1,18 @@
+[id="eda-delete-rulebook-activations-with-cont-tokens"]
+
+= Deleting rulebook activations associated with controller tokens
+
+After deleting your controller tokens, you must delete the rulebook activations that were associated with them.
+
+.Prerequisites
+* You have deleted your previous controller tokens. 
+* You have identified which rulebook activations are associated with the deleted controller tokens. 
```

Comment:
Hi @zkayyali812 and Joe P.  I was curious how the end user would know which rulebook activations were associated with a rulebook activation. I added these prereqs. Let me know if these are correct.

Reply:
Currently I dont think we have a way for this to be displayed effectively to a user. Perhaps we can ask the team about this further to double check here just in case.
Regarding switching the order, I think that may be a good idea, but open to other perspectives on this as well.